website/views/options/social/group.jade

```diff
@@ -82,7 +82,7 @@ a.pull-right.gem-wallet(ng-if='group.type!="party"', popover-trigger='mouseenter
                     span.glyphicon.glyphicon-ban-circle(tooltip=env.t('banTip'))
                 a.media-body
                   span(ng-click='clickMember(member._id, true)')
-                    | {{member.profile.name}}
+                    | {{member.profile.name}} (<strong>{{member.stats.hp.toFixed(1)}}</strong> #{env.t('hp')})
```

Comment:
I'm pretty sure the <strong> tags could be replaced with just `strong {{member.stats.hp.toFixed(1)}}`
That would make the source code neater. Could you try that?